src/Nancy.Tests.Functional/Tests/CookieTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace Nancy.Tests.Functional.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Bootstrapper;
+    using ModelBinding;
+
+    using Nancy.Cookies;
+
+    using Testing;
+    using Xunit;
+
+    public class CookieTestsFixture

```

Comment:
@richardprice Can you either 1) rename the class, or 2) rename the file? :smile: